tests/console/ruby.pm

```diff
@@ -0,0 +1,39 @@
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Package: ruby
+# Summary: -  Install ruby package
+#          -  run a simple ruby program
+# Maintainer: QE Core <qe-core@suse.de>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils;
+
+sub run {
+    select_serial_terminal;
+    my $ruby = script_output("zypper se -s ruby | awk -F '|' '{ print \$2 }' | grep -oE 'ruby[0-9|.]+' | sort -V | tail -n 1");
```

Comment:
Can you add some description here why you specify a ruby version?

Reply:
Here does not specify a ruby version, it's used to find the latest version.
For example on 15-SP7, ruby-2.x is installed by default, but ruby-3.x is available, so find out the newer version and install it.

Reply:
Good to know, so can we run the tests for all available versions like we did for python3?

Reply:
For 15-SP7, there are `ruby2.5` and `ruby3.4` I will try to test for all versions.

Reply:
Done. From the VRs, there is only 15-SP7 have two ruby version `2.5` and `3.4`. Others (such as 15-SP6) only have one version 2.x